docs/legal/assets/site.css

```
@@ -0,0 +1,646 @@
+  --text: #1a2332;
+  --text-muted: #5c6778;
+  --accent: #0d7a7a;
+  --accent-hover: #0a6363;
+  --accent-soft: rgba(13, 122, 122, 0.1);
+  --border: rgba(26, 35, 50, 0.1);
+  --shadow: 0 4px 24px rgba(26, 35, 50, 0.08);
+  --radius: 12px;
+  --radius-lg: 20px;
+  --max-width: 72rem;
+  --content-width: 42rem;
+  --font: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)